collector/mongodb_collector.go

```diff
@@ -37,6 +37,8 @@ type MongodbCollectorOpts struct {
 	CollectCollectionMetrics bool
 	CollectProfileMetrics    bool
 	CollectConnPoolStats     bool
+	CollectParameterMetrics  bool
+	CollectParameters        string
```

Comment:
Unless we have a default set, we can use just the string

Reply:
the default set is cursorTimeoutMillis